### DISCUSSION TITLE
how to build a websocket route that ignores client side message and send messages to client each second?

### DISCUSSION THREAD

**Discussion by DennisZhangOiler** [ntex-rs/ntex#476] [Q&A]
Aslo, how can I get state in a websocket route?

**Comment by 0xle0ne** [comment]
You can browse the example there: https://github.com/ntex-rs/examples/tree/master/websocket

**Reply by DennisZhangOiler** [reply]
the low level one was too complicated and with fn factory, you have to set None to all client request.
